kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -315,7 +315,7 @@ public void convert(boolean heatCoolEq, String prelude, StringBuilder semantics,
         semantics.append("\n// priority groups\n");
         genPriorityGroups(priorityList, priorityToPreviousGroup, priorityToAlias, topCellSortStr, semantics);
         semantics.append("endmodule ");
-        convert(attributes, module.att(), semantics, null, null);
+        convert(attributes, module.att().remove(Att.DIGEST()), semantics, null, null);
```

Comment:
instead of doing this, let's just remove the digest attribute when converting from a FlatModule back to a Module.

Reply:
That's not going to work for kprove.
We store `CompileDefinition.parsedDefinition` which has `Module`.
If we strip that attribute we won't have it when we do `slurp` in kprove. Which is where the failure happens.